runtime/doc/options.txt

```diff
@@ -8451,34 +8450,42 @@ A jump table for the options with a short description can be found at |Q_op|.
 	|           |
 <
 
-
-					*'tabsidebaralign'* *'tsba'*
-					*'notabsidebaralign'* *'notsba'*
-'tabsidebaralign' 'tsba'		boolean	(default off)
+					*'tabpanelopts* *'tplo'*
+'tabpanelopts' 'tplo'		string	(default "")
```

Comment:
'tabpanelopt'. Not 'tabpanelopts'.
Currently, there are many option names ending in opt, but no option names ending in opts.

```
:h *opt'@en<C-D>
'mopt'            'culopt'          'completeopt'     'breakindentopt'
'popt'            'diffopt'         'messagesopt'
'briopt'          'scrollopt'       'cursorlineopt'
:h *opts'@en<C-D>
:h *opts'@en
```